.gitignore

```diff
@@ -14,4 +14,5 @@ test-output/
 /bin/
 *.log
 config/
-rawlsdb/
\ No newline at end of file
+rawlsdb/
+*.pid
```

Comment:
As you prefer, we haven't given it its own PR in the other repos but we certainly could here.